#File Transfer
string path
---
eros/file[] files

